src/components/input_fields/__tests__/search_field.test.tsx

```diff
@@ -50,9 +50,12 @@ describe('The search field component', () => {
         props.staticField = true;
         props.children = 'a child';
         wrapper.setProps(props);
-        props['inputChildren'] = props['children'];
+        // This test needs to be rewritten
```

Comment:
Is that possible to elaborate the reason why this test needs to be rewritten?

Reply:
I think it's not a very well written test. The whole file is a bit meh. In the `beforeEach` there's even an assertion… 